dfuse-io/client-js
# add firehose type

Overview:

This PR adds type support for firehose blocks. Inspiration is to create a forked eosq instance that just displays transactions pulled from the firehose gRPC stream.

To do:

- Test on local eosq instance - done
- Ensure no additional fields required - done
- Ensure optional fields are included - done

Please let me know if there's other aspects that should be added while I test and update.
